### PR TITLE
Document API endpoints

### PR DESCRIPTION
**Pull request by dbrgn**
Without the docs, it always takes 10 minutes for me to find out how to call the endpoints :see_no_evil: 

I updated the README with all the endpoints I could figure out with my incomplete Rails knowledge. If there are others, it would be great if you could add them too, @schmijos!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,29 @@
...
-## API Endpoints
-
-Base path is `/api/`.
-
-- `/sensors` (full CRUD)
-- `/measurements` (full CRUD)
-- `/sponsors` (full CRUD)
+- `/api/mobile_app/sensors`
+- `/api/mobile_app/sensors/<sensor-id>/sponsor`
+- `/api/mobile_app/sensors/<sensor-id>/daily_temperatures`
```
Comment (Makram95) [inline README.md:25]: Maybe also add the possible params from, to & limit here

Comment (Makram95) [reply README.md:25]: of course for /daily_temperatures and /hourly_temperatures instead of /sponsor

Comment (dbrgn) [reply README.md:25]: Ah good point, I wanted to add parameters as well, but then forgot :slightly_smiling_face:

**Comment by Makram95** [comment]
When requesting all sensors with `/api/mobile_api/sensors/' the sensor entries are missing the max/min/avg temperatures, is this intended?
@dbrgn @schmijos

**Comment by schmijos** [comment]
Thank you. I agree that this is needed. I'm currently experimenting with _rswag_ at work. Maybe I'll introduce that here.

In the meantime you can get all the routes by running `bundle exec rails routes` in the console:

```
                               Prefix Verb   URI Pattern                                               Controller#Action
                                 root GET    /                                                         home#index
              aggregated_measurements GET    /api/measurements/aggregated(.:format)                    measurements#aggregated {:format=>"json"}
                         measurements GET    /api/measurements(.:format)                               measurements#index {:format=>"json"}
                                      POST   /api/measurements(.:format)                               measurements#create {:format=>"json"}
                          measurement GET    /api/measurements/:id(.:format)                           measurements#show {:format=>"json"}
                                      PATCH  /api/measurements/:id(.:format)                           measurements#update {:format=>"json"}
                                      PUT    /api/measurements/:id(.:format)                           measurements#update {:format=>"json"}
                                      DELETE /api/measurements/:id(.:format)                           measurements#destroy {:format=>"json"}
                              sensors GET    /api/sensors(.:format)                                    sensors#index {:format=>"json"}
                                      POST   /api/sensors(.:format)                                    sensors#create {:format=>"json"}
                               sensor GET    /api/sensors/:id(.:format)                                sensors#show {:format=>"json"}
                                      PATCH  /api/sensors/:id(.:format)                                sensors#update {:format=>"json"}
                                      PUT    /api/sensors/:id(.:format)                                sensors#update {:format=>"json"}
                                      DELETE /api/sensors/:id(.:format)                                sensors#destroy {:format=>"json"}
                             sponsors GET    /api/sponsors(.:format)                                   sponsors#index {:format=>"json"}
                                      POST   /api/sponsors(.:format)                                   sponsors#create {:format=>"json"}
                              sponsor GET    /api/sponsors/:id(.:format)                               sponsors#show {:format=>"json"}
                                      PATCH  /api/sponsors/:id(.:format)                               sponsors#update {:format=>"json"}
                                      PUT    /api/sponsors/:id(.:format)                               sponsors#update {:format=>"json"}
                                      DELETE /api/sponsors/:id(.:format)                               sponsors#destroy {:format=>"json"}
 daily_temperatures_mobile_app_sensor GET    /api/mobile_app/sensors/:id/daily_temperatures(.:format)  mobile_app/sensors#daily_temperatures {:format=>"json"}
hourly_temperatures_mobile_app_sensor GET    /api/mobile_app/sensors/:id/hourly_temperatures(.:format) mobile_app/sensors#hourly_temperatures {:format=>"json"}
            sponsor_mobile_app_sensor GET    /api/mobile_app/sensors/:id/sponsor(.:format)             mobile_app/sensors#sponsor {:format=>"json"}
                   mobile_app_sensors GET    /api/mobile_app/sensors(.:format)                         mobile_app/sensors#index {:format=>"json"}
                    mobile_app_sensor GET    /api/mobile_app/sensors/:id(.:format)                     mobile_app/sensors#show {:format=>"json"}
```

**Comment by schmijos** [comment]
> When requesting all sensors with `/api/mobile_app/sensors/' the sensor entries are missing the max/min/avg temperatures, is this intended?
> @dbrgn @schmijos
> 
> edit: plus the url field is still there

Yes, this was intended. I thought this was information meant for the detail page. Do you explicitly want everything in the index? In that case I'll completely remove the sensor detail page.

**Review by schmijos — APPROVED** [review]
🙏 thank you

**Comment by dbrgn** [comment]
> It's more an architectural decision to refer to the detail page for expensive information. Do you explicitly want everything in the index?

How expensive are the aggregations? If n additional aggregation queries are required for every sensor, then it's probably better to only calculate these aggregations when requesting the details, not in the index.

Or, from another perspective: Do we need this information when showing an overview of all sensors, or is the current temperature sufficient?

(If the aggregations can be calculated relatively cheaply for all sensors using a join, then it should be fine to include all information in the index response.)

**Comment by dbrgn** [comment]
> In the meantime you can get all the routes by running bundle exec rails routes in the console:

Ah, that's great to learn :slightly_smiling_face:

**Comment by Makram95** [comment]
@schmijos It's not that necessary to include them. I now think it makes more sense to leave them out and just make a specific specific sensor request when opening a detail view anyway.

Would it also be possible to also include the time of the last_temperature when requesting a sensor directly?

**Pull request merged**